src/NuGet.Status/NuGet.Status.csproj

```diff
@@ -122,6 +122,7 @@
     <Content Include="NuGetGallery\src\Bootstrap\dist\css\bootstrap.css" />
     <Content Include="NuGetGallery\src\Bootstrap\dist\css\bootstrap-theme.css" />
     <Content Include="NuGetGallery\src\NuGetGallery\Views\**" />
+    <Content Remove="NuGetGallery\src\nugetcdnredirect\web.config" />
```

Comment:
How is this being included in the first place?

Reply:
Not sure, we updated gallery submodule with latest version, which is after we merged all repos. before it was just gallery repo. I thought it was repos merge thing. But looked at original Gallery, looks like cdnredirect exist already before merging?

Reply:
@lyndaidaii 
Do we need anymore update here?